libs/documentation/src/lib/storybook/actions/actions.stories.ts

```diff
@@ -151,4 +178,4 @@ export const Introduction: Story = (args) => ({
 });
 Introduction.parameters = { options: { showPanel: false } };
 
-export const __namedExportsOrder = ['Introduction', 'Configurable', 'ModelTrigger', 'Size'];
+export const __namedExportsOrder = ['Introduction', 'Configurable', 'ModelTrigger', 'Size', 'Label'];
```

Comment:
Only introduction and Configurable should be out of alphabetic order. Unfortunately, there ins't a way to tell this to Storybook so we've got to manually make sure the order is right. Stick label before modelTrigger.

Reply:
changed to alphabetical order